### PR TITLE
Numpy square

### PR DESCRIPTION
**Pull request by cmaxreilly**
### Description

Added an entry for the numpy.square() function. Fixed some small formatting errors in a c file.

### Issue Solved

#5701 

### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/numpy/concepts/math-methods/terms/square/square.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  - 'paths/computer-science'
+---
+
+
+In NumPy, the **`.square()`** method is used to calculate the square of a number or the square of
+the elements of an array. It is employed in mathematical computations common in machine learning,
+data analysis, engineering, and graphics.
+
+## General Syntax
+```python
```
[the remark this answers is not in the document]
Comment (cmaxreilly) [reply content/numpy/concepts/math-methods/terms/square/square.md:21]: Okay. Is this true for all of my syntax blocks?

**Comment by cmaxreilly** [comment]
@dakshdeepHERE, I believe that I have made all of the changes you requested. Please let me know if I have missed anything!

**Review by mamtawardhani — COMMENTED** [review]
Thank you for contributing to Codecademy Docs @cmaxreilly  😄 

The entry looks good to be merged! 🚀

**Review by mamtawardhani — APPROVED** [review]
Thank you for contributing to Codecademy Docs @cmaxreilly  😄 

The entry looks good to be merged! 🚀

**Pull request merged**